### ISSUE TITLE
Other Training datasets and form of import dataset

### ISSUE THREAD

**Issue by bioguy2018** [pcahan1/singleCellNet#27]
Dear all,
Are there any other comprehensive mouse training datasets that you might be aware of?
Also, so as far as I understood any sc dataset that has been analysed can be imported as a training set and I just need the expression and  the cluster annotation right?
Also I am wondering should my inquiry expression be raw data or is it better to have a normalized, scaled and in general a processed data?

Thank you!

**Comment by yuqiyuqitan** [comment]
Hi,
I have uploaded some sc atlas training datasets at the bottom of SCN README. Commonly ones are Tabula Muris, and MicroWellSeq. Other than those, you can find your own as well provided it has cell type annotation and expression data. 

As for your query data, in my experience, raw or processed both work fine. 
Best
Yuqi

**Comment by bioguy2018** [comment]
Thank you!
